tests/test_automatically_annotate_mass2motifs/test_mass2motif.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from automatically_annotate_mass2motifs.mass2motif import Mass2Motif
+
+
+def test_creating_mass2motif():
+    words = ['fragment_375.2225',
+             'loss_80.0275',
+             'loss_128.0625']
+    probabilities = [0.0055,
+                     0.0012,
+                     0.0019]
+    mass2motif = Mass2Motif(words, probabilities)
+    assert mass2motif.fragments.mz == [375.2225]
+    assert np.all(mass2motif.losses.mz == [80.0275, 128.0625])
+    assert mass2motif.fragments.intensities == [0.0055]
+    assert np.all(mass2motif.losses.intensities == [0.0012, 0.0019])
+
+    assert mass2motif.bin_size == 0.005, "bin_size is not correctly set"
+
+def test_creating_unordered_mass2motif():
+    words = ['fragment_375.2225',
+             'loss_128.0625',
+             'loss_80.0275']
+    probabilities = [0.0055,
+                     0.0019,
+                     0.0012,]
```

Comment:
```suggestion
def test_creating_unordered_mass2motif(words, probabilities):
```

Reply:
Just to reduce code duplication (should then also be done for the following tests).